libs/island-ui/core/src/lib/IconRC/icons/UploadOutline.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import type { SvgProps as SVGRProps } from '../types'
+
+const SvgUploadOutline = ({
+  title,
+  titleId,
+  ...props
+}: React.SVGProps<SVGSVGElement> & SVGRProps) => {
+  return (
+    <svg
+      className="download-outline_svg__ionicon"
+      viewBox="0 0 512 512"
+      aria-labelledby={titleId}
+      transform="matrix(1,0,0,-1,0,0)"
```

Comment:
this has transform matrix and the other is rotate 180, why is that 💭 ?

Reply:
coderabbit review fix mixup 🙈